NEWS

```diff
@@ -925,7 +925,7 @@
       o Overhauled Win32 builds
       o Cleanups and fixes to the Big Number (BN) library
       o Support for ASN.1 GeneralizedTime
-      o Splitted ASN.1 SETs from SEQUENCEs
+      o Split ASN.1 SETs from SEQUENCEs
```

Comment:
The NEWS items should remain as they are. Please revert this change.

Reply:
I have reverted the NEWS file's change.